runtests.py

```diff
@@ -13,20 +15,30 @@
 
 
 def main():
+    """Parse arguments and run main test suite."""
     parser = ArgumentParser(description='Run the dj-stripe Test Suite.')
     parser.add_argument("--skip-utc", action="store_true", help="Skip any tests that require the system timezone to be in UTC.")
     parser.add_argument("--no-coverage", action="store_true", help="Disable checking for 100% code coverage (Not advised).")
     parser.add_argument("--no-pep8", action="store_true", help="Disable checking for pep8 errors (Not advised).")
+    parser.add_argument("--no-pydocstyle", action="store_true", help="Disable checking for pydocstyle errors (Not advised).")
```

Comment:
Make sure to add this to the docs. I don't remember if the othes are covered in the docs. If they aren't, nevermind.